.github/workflows/create-release-pr.yml

```
@@ -0,0 +1,138 @@
+        run: cargo build --release
+
+      - name: Sync versions across package.json files
+        run: node scripts/sync-versions.js sync
+
+      - name: Commit changes
+        run: |
+          VERSION="${{ github.event.inputs.version }}"
+
+          # Add all modified files
+          git add Cargo.toml Cargo.lock packages/@mocks-rs/*/package.json CHANGELOG.md
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)